### ISSUE TITLE
(Eurostat data) Find a way to associate metadata to each data of an array

### ISSUE THREAD

**Issue by alixdamman** [larray-project/larray#689]
@gdementen One of our team specifically asked for this feature. We need more information about how they want to use these metadata before to start any implementation work on this feature.

**Comment by gdementen** [comment]
To each data? You mean for each cell? I guess this is to contain the "letters" present in some eurostat datasets (provisional etc). Currently we could have an object array next to the main array containing the letters. Or maybe it would be better in this particular case to have several boolean arrays.

We will probably want to add native support for attaching such (an) array(s) to the main array. eg.
arr.cellmeta
or arr.cellmeta.provisional

**Comment by gdementen** [comment]
Yes, that would be somewhat similar to masked_array except there could be several masks. We indeed need more information. As a side note, I think 0.30 is too early for this feature...
